docs/pages/_app.js

```diff
@@ -0,0 +1,3 @@
+import MyApp from '@material-ui/monorepo/docs/pages/_app';
+
+export default MyApp;
```

Comment:
The beauty of the approach

Reply:
How about`export {default as MyApp} from...`? 1 line

Reply:
As far as I know, named export doesn't work with Next.js custom App: https://nextjs.org/docs/advanced-features/custom-app. 

We could probably do

```jsx
export { default } from '@material-ui/monorepo/docs/pages/_app';
```

But the commit the comment is based on is outdated, the file has grown to:

```jsx
import MyApp from '@material-ui/monorepo/docs/pages/_app';
import { LicenseInfo } from '@material-ui/x-grid';

// Remove the license warning from demonstration purposes
LicenseInfo.setLicenseKey(
  '0f94d8b65161817ca5d7f7af8ac2f042T1JERVI6TVVJLVN0b3J5Ym9vayxFWFBJUlk9MTY1NDg1ODc1MzU1MCxLRVlWRVJTSU9OPTE=',
);

export default MyApp;
```